service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/ObservationStatementMapper.java

```diff
@@ -114,4 +139,14 @@ private String prepareCode(Observation observation) {
     private boolean isRangeUnitValid(String unit, Quantity quantity) {
         return quantity.hasUnit() && !unit.equals(quantity.getUnit());
     }
+
+    private boolean isInterpretationCode(CodeableConcept interpretation) {
+        Coding coding = interpretation.getCodingFirstRep();
+        String codingSystem = coding.getSystem();
+        String code = coding.getCode();
+
+        return (coding.hasSystem() && codingSystem.equals("http://hl7.org/fhir/v2/0078"))
+            && (code.equals("H") || code.equals("HH") || code.equals("HU") || code.equals("L") || code.equals("LL")
+            || code.equals("LU") || code.equals("A") || code.equals("AA"));
```

Comment:
This could potentially be easier to read and maintain if you use an inline set
```suggestion
            && Set.of("H", "HH", "HU", "L", "LL", "LU", "A", "AA").contains(code));
```

Reply:
Agreed & done in d69f313